package.json

```diff
@@ -1,56 +1,55 @@
 {
   "name": "reoserv.net",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "",
   "type": "module",
   "scripts": {
     "build": "remix vite:build",
     "dev": "node ./server.js",
-    "lint": "biome check --apply-unsafe .",
+    "lint": "biome check --write --unsafe .",
     "lint:docs": "prettier --write *.md ./content/**/*.md",
     "start": "cross-env NODE_ENV=production node ./server.js"
   },
   "keywords": ["reoserv", "reoserv.net", "endless online emulator"],
   "author": "Richard Leek <richard@richardleek.com>",
   "license": "MIT",
   "dependencies": {
-    "@remix-run/express": "^2.8.1",
-    "@remix-run/node": "^2.8.1",
-    "@remix-run/react": "^2.8.1",
+    "@remix-run/express": "^2.10.0",
```

Comment:
No new future flags have been enabled. 

"Fog Of War" and "Single Fetch" are both unstable right now.